app/gitcoinbot/tests.py

```diff
@@ -21,32 +21,39 @@
 from django.test import TestCase
 
 from gitcoinbot.actions import (
-    claim_bounty_text, confused_text, help_text, new_bounty_text, new_tip_text, parse_comment_amount,
-    parse_tippee_username,
+    confused_text, get_text_from_query_responses, help_text, new_bounty_text, new_tip_text, parse_comment_amount,
+    parse_tippee_username, start_work_text, submit_work_or_new_bounty_text, submit_work_text,
 )
+from gitcoinbot.models import GitcoinBotResponses
 
 
 class gitcoinbotActions(TestCase):
     """Define the Gitcoin Bot actions TestCase."""
 
     def test_help_text(self):
         """Test that help_text function returns the correct text."""
+        self.maxDiff = None
```

Comment:
This doesn't appear to be used.

Reply:
I add it because when the test failed only show a shortened version of the strings, so it was difficult know how the strings are different.